feature/aft/aft_base/otg_tests/aft_base/README.md

```diff
@@ -0,0 +1,140 @@
+# AFT-1.1: AFTs Base
+
+## Summary
+
+IPv4/IPv6 unicast routes next hop group and next hop.
+
+## Procedure
+
+Configure DUT:port1,port2,port3 for IS-IS session with ATE:port1,port2,port3
+*   IS-IS must be level 2 only with wide metric.
+*   IS-IS must be point to point.
+*   Send 1000 ipv4 and 1000 ipv6 IS-IS prefixes from ATE:port3 to DUT:port3.
+*   IS-IS must be up and running, prefixes must be in RIB and FIB.
+
+Establish eBGP sessions between ATE:port1,port2 and DUT:port1,port2 and another between ATE:port3 and DUT:port3.
+*   Configure eBGP over the interface ip.
+*   eBGP must be multipath.
+*   Advertise 1000 ipv4,ipv6 prefixes from ATE port1,port2 observe received prefixes at DUT.
+*   Validate total number of entries of AFT for IPv4 and IPv6.
+*   Each prefix must have 2 next hops pointing to ATE port1,port2.
+*   Advertise 100 ipv4,ipv6 from ATE port3 observe received prefixes at DUT.
```

Comment:
What is the hope here? Are we testing gNMI updates here? Should we be concurrently handling a gRPC stream with a subscription (ON_CHANGE, for example) to make sure we get these 100 prefixes in addition to the previous 1,000?

Or should this be in the next section where we would like these to be the destination at the end of the tunnels?

Reply:
done

Reply:
Can you explicitly list out which of the options I suggested you are doing here?

Reply:
done. Please refer the latest verification section